packages/terra-menu/src/Menu.module.scss

```diff
@@ -46,6 +46,10 @@
       word-wrap: normal;
     }
 
+    .main-header-title {
+      margin-left: var(--terra-menu-mainmenu-text-margin-left, 10px);
```

Comment:
We'll want a consistent approach to naming these custom properties. Following other conventions in this file I would expect this to be something similar to:

```
--terra-menu-header-title-margin-left
```

There is another header-title property in this file, we'll want to make sure these don't collide.

Reply:
We'll want to get UX feedback on the units being used here. Generally margins are represented with `rem` units, but there seems to be some inconsistency in this file.

Reply:
> We'll want a consistent approach to naming these custom properties. Following other conventions in this file I would expect this to be something similar to:
> 
> ```
> --terra-menu-header-title-margin-left
> ```
> 
> There is another header-title property in this file, we'll want to make sure these don't collide.

Updater here d0fb28990d97c747f039a532a4922640215c05a7

Reply:
> We'll want to get UX feedback on the units being used here. Generally margins are represented with `rem` units, but there seems to be some inconsistency in this file.

@neilpfeiffer @jeremyfuksa